sui_programmability/examples/README.md

```diff
@@ -1,6 +1,7 @@
 Lots of Move code examples, partitioned by category:
 
 * basics: The very simplest examples of Sui programming.
+* crypto: A simple contract to perform verification and ecrecover.
```

Comment:
replace with _crypto: A simple contract to perform ECDSA secp256k1 signature verification and ecrecover (derive the public key from a signature)._

Reply:
Thanks, Kostas! Done.